docs/README.rst

```diff
@@ -166,6 +166,22 @@ You can use nat for interface. This is supported for IPv4 alone. IPv6 deployment
           - 10.20.0.2
 ```
 
+Configure the firewall using `tables`
+-------------------------------------
+
+The state `iptables.tables` let's you configure your firewall iterating over pillars
+defining rules and policies to add to the different tables (filter, mangle, nat) instead of using services.
+This way, you can configure iptables the *classic way*. Note that you still need to include the `iptables` state.
+
+To enable the 'tables' mode, set:
+
+```yaml
+firewall:
+  use_tables: True
+```
```

Comment:
It's not pleasant switching between `.md` and `.rst`!

```rst
.. code-block:: yaml

   firewall:
     use_tables: True
```

Reply:
By the way, I've just noticed that there are many places this needs to be done within the `README` -- could you do those as well?

Reply:
Done